src/visualizer.js

```diff
@@ -64,7 +64,7 @@ if (config.visualizer.enabled) {
           });
         }
         // draw creep positions
-        if (config.visualizer.showCreeps) {
+        if (config.visualizer.showCreeps && room.memory.position) {
```

Comment:
sure we need these? if vusual runs on a room the positions whould already be in place?